test/suites/repo_client/docker-compose.override.yml

```diff
@@ -9,7 +9,7 @@ services:
       - ./repo-extra-install.sh:/extra-install.sh
 
   wikibase-client:
-    image: ${WIKIBASE_SUITE_WIKIBASE_IMAGE_URL}
+    image: wikibase/wikibase
```

Comment:
I am not against hardcoding this, but then we would remove all the `WBS_*_IMAGE_URL` things from the `build.env` files, right?

Reply:
Maybe. There are a couple things going on here, but for now this was just me making something consistent within the test setup. They were mostly all hardcoded except these. This was from the earlier PR with a note about it I think...